server/chat-plugins/lottery.js

```diff
@@ -0,0 +1,258 @@
+'use strict';
+
+const LOTTERY_FILE = 'config/chat-plugins/lottery.json';
+
+/** @type {typeof import('../../lib/fs').FS} */
+const FS = require(/** @type {any} */('../../.lib-dist/fs')).FS;
+
+/** @type {{[roomid: string]: {maxWinners: number, name: string, markup: string, participants: {[ip: string]: string}}}} */
+let lotteries = Object.create(null);
+try {
+	lotteries = require(`../../${LOTTERY_FILE}`);
```

Comment:
you need to modify the prototype of this to make sure it doesn't have `constructor` and the like

Reply:
The lottery loading was a bit verbose so I just re-wrote it.

I also don't really understand what you were saying, so can you check if this is still a problem in the new system?